diff --git a/plugins/icons_papirus/change_color.sh b/plugins/icons_papirus/change_color.sh
index ecbc1e6a..80530c80 100755
--- a/plugins/icons_papirus/change_color.sh
+++ b/plugins/icons_papirus/change_color.sh
@@ -59,6 +59,10 @@ do
 			ICONS_COLOR="${2#\#}"  # remove leading hash symbol
 			shift
 			;;
+		-t|--target-dir)
+			DEST_PATH_ROOT="$2"
+			shift
+		;;
 		-*)
 			echo "unknown option $1"
 			print_usage 2
@@ -99,7 +103,7 @@ trap post_clean_up EXIT SIGHUP SIGINT SIGTERM
 : "${ICONS_COLOR:=$SEL_BG}"
 : "${OUTPUT_THEME_NAME:=oomox-$THEME}"
 
-output_dir="$HOME/.icons/$OUTPUT_THEME_NAME"
+output_dir="${DEST_PATH_ROOT-$HOME/.icons}/$OUTPUT_THEME_NAME"
 
 light_folder_fallback="$ICONS_COLOR"
 medium_base_fallback="$(darker "$ICONS_COLOR" 20)"
