diff --git a/plugins/icons_papirus/change_color.sh b/plugins/icons_papirus/change_color.sh
index 9c4ecf57..d35e599a 100755
--- a/plugins/icons_papirus/change_color.sh
+++ b/plugins/icons_papirus/change_color.sh
@@ -118,6 +118,7 @@ dark_stroke_fallback="$(darker "$ICONS_COLOR" 56)"
 
 echo ":: Copying theme template..."
 cp -R "$root/papirus-icon-theme/Papirus" "$tmp_dir/"
+chmod -R +w "$tmp_dir/Papirus"
 echo "== Template was copied to $tmp_dir"
 
 
